app/src/main/res/values/strings.xml

```diff
@@ -20,17 +20,17 @@
     <!-- Long click options -->
     <string name="long_click_title">What do you want to do?</string>
     <string name="long_click_open">Open \"%1s\"</string>
-    <string name="long_click_store">See its store page</string>
+    <string name="long_click_store">View store page</string>
     <string name="long_click_settings">Access its settings</string>
     <string name="long_click_remove_shortcut">Remove this shortcut</string>
 
     <!-- About -->
     <string name="app_name">Discreet Launcher</string>
     <string name="author" translatable="false">Vincent Falzon</string>
-    <string name="licence_name">GNU General Public License v3.0</string>
+    <string name="licence_name">GPLv3+</string>

```

Comment:
Although it is subjective, I find the license name more elegant in the current format (indeed a bit longer).

Regarding the license itself, it is not planned to switch from GPLv3 to GPLv3+ because I am not comfortable committing to texts that do not exist yet (future GPLv4, GPLv5, etc.).

Reply:
```suggestion
    <string name="licence_name">GNU General Public License v3.0</string>

```

Reply:
It is a bit awkward to either translate "license" or not. Either it looks wrong, or it is less strictly correct.